dom.bs

```diff
@@ -942,25 +942,49 @@ method must, when invoked, run these steps:
 
 <h3 id=constructing-events>Constructing events</h3>
 
-When a <dfn export for=Event id=concept-event-constructor>constructor</dfn> of the
-{{Event}} interface, or of an interface that inherits from the {{Event}} interface, is
-invoked, these steps must be run:
+<p>When a <dfn export for=Event id=concept-event-constructor>constructor</dfn> of the {{Event}}
+interface, or of an interface that inherits from the {{Event}} interface, is invoked, these steps
+must be run, given the arguments <var>type</var> and the optional <var>eventInitDict</var>:
 
 <ol>
- <li><p>Create an <a>event</a> that uses the interface the constructor was invoked upon.
+ <li><p>Create a new object <var>event</var> using this interface.
 
- <li><p>Set its <a>initialized flag</a>.
+ <li><p>Set <var>event</var>'s <a>initialized flag</a>.
 
- <li><p>Initialize the {{Event/type}} attribute to the <var>type</var> argument.
+ <li><p>Initialize <var>event</var>'s {{Event/type}} attribute to <var>type</var>.
 
- <li><p>If there is an <var>eventInitDict</var> argument, then for each <a>dictionary member</a>
- present, find the attribute on <a>event</a> whose <a spec=webidl>identifier</a> matches the key of
- the <a>dictionary member</a> and then set the attribute to the value of that
- <a>dictionary member</a>.
+ <li><p>If <var>eventInitDict</var> is given, then for each <a>dictionary member</a> present, find
+ the attribute on <var>event</var> whose <a spec=webidl>identifier</a> matches the key of the
+ <a>dictionary member</a> and then set the attribute to the value of that <a>dictionary member</a>.
 
- <li><p>Return the <a>event</a>.
+ <li><p>Return <var>event</var>.
+</ol>
+
+<p>To <dfn export id=concept-event-create lt="creating an event|create an event">create an
+event</dfn> using <var>eventInterface</var>, which must be either {{Event}} or an interface that
+inherits from it, run these steps:</p>
+
+<ol>
+ <li><p>Create a new object <var>event</var> using the <var>eventInterface</var> interface.
+
+ <li><p>Set <var>event</var>'s <a>initialized flag</a>.
+
+ <li><p>For each <a>dictionary member</a> of the dictionary accepted by <var>eventInterface</var>'s
+ constructor (which will either be {{EventInit}} or a dictionary that inherits from it), find the
+ attribute on <var>event</var> whose <a spec=webidl>identifier</a> matches the key of the
+ <a>dictionary member</a> and then set the attribute to the default value of that <a>dictionary
+ member</a>.
```

Comment:
@heycam we can set type separately quite easily (that's a constant for all events). Though if you have other ideas that's fine too. I'll let @domenic comment on your idea. I think I'm happy as long as you are all happy.